lib/features/modeling/behavior/AdaptiveLabelPositioningBehavior.js

```diff
@@ -122,6 +122,10 @@ export default function AdaptiveLabelPositioningBehavior(eventBus, modeling) {
 
     var elementTrbl = asTRBL(element);
 
+    if (Object.values(elementTrbl).some(value => value == undefined || isNaN(value))) {
```

Comment:
Is this the root cause of the issue, or do we workaround a bug here?

Please add a test case to verify the behavior. I suggest you to do this as a start (to also guide your root causing efforts).

Reply:
The issue was that the flow element doesn't have top bottom coordinates which was returning NaN when updating the position of the element. I'll add test case for this.